packages/amplify-category-geo/src/provider-controllers/import.ts

```diff
@@ -0,0 +1,141 @@
+import { readFileSync } from 'fs-extra';
+import { $TSContext } from 'amplify-cli-core';
+import { prompter, printer } from 'amplify-prompts';
+import { existsSync, writeFileSync } from 'fs-extra';
+import { join } from 'path';
+import aws from 'aws-sdk';
+import { ServiceName } from '../service-utils/constants';
+import { validateGeoJSONObj } from '../service-utils/validateGeoJSONObj';
+import {
+  FeatureCollection, ImportParams, GeofenceCollectionParams, GeofenceParams, IdentifierOption, IdentifierInfo,
+} from '../service-utils/importParams';
+import { getGeoResources, getGeoServiceMeta } from '../service-utils/resourceUtils';
+
+const MAX_ENTRIES_PER_BATCH = 10;
+
+export const importResource = async (context: $TSContext) => {
+  const geofenceCollectionResourcesMap = await getGeoServiceMeta(ServiceName.GeofenceCollection);
+  const collectionNames = await getGeoResources(ServiceName.GeofenceCollection);
+  if (collectionNames.length === 0) {
+    throw new Error('Geofence collection is not found. Run `amplify geo add` to create a new geofence collection.')
+  }
+  // Get provisioned geofence collection name
+  const provisionedCollectionNames = collectionNames.filter(collectionName => {
+    const collection = geofenceCollectionResourcesMap[collectionName];
+    if (!collection.output || !collection.output.Name || !collection.output.Region) {
+      return false;
+    }
+    return true;
+  }).map(collectionName => geofenceCollectionResourcesMap[collectionName].output.Name);
+  if (provisionedCollectionNames.length === 0) {
+    throw new Error('No geofence is not provisioned yet. Run \`amplify push\` to provision geofence collection.')
+  }
+  // Get collection region
+  const collectionRegion = geofenceCollectionResourcesMap[collectionNames[0]].output.Region;
```

Comment:
Only the provisioned collections will have the Region output. Have to account for that.